source/kit/components/ui/components.ui.config.js

```diff
@@ -21,4 +21,25 @@ angular.module('msm.components.ui')
 .config(function(uibDatepickerPopupConfig) {
   uibDatepickerPopupConfig.showButtonBar = false;
   uibDatepickerPopupConfig.datepickerPopupTemplateUrl = 'components/ui/msm-datepicker/msm-datepicker-popup.html';
+})
+
+.constant('uiDatetimePickerConfig', {
```

Comment:
Is it possible to customize these settings in a project? I think it would be important that developers can use custom settings that may be different from the ones used here.